core/v0/api/core.yaml

```diff
@@ -1764,30 +1764,9 @@ components:
           $ref: '#/components/schemas/DecimalValue'
       
     Name:
-      description: Describes the name of a person
-      type: object
-      properties:
-        full:
-          type: string
-          description: 'Format: ./{honorific prefix}/{given_name}/{family_name}/{additional_name}/{call_sign}/{honorific_suffix}'
-        additional_name:
-          type: string
-          description: 'An additional name for a Person, can be used for a middle name.'
-        family_name:
-          type: string
-          description: 'Family name. In India, it is the last name of an Person. This can be used along with givenName instead of the name property.'
-        given_name:
-          type: string
-          description: 'Given name. In India, it is the first name of a Person. This can be used along with familyName instead of the name property.'
-        call_sign:
-          type: string
-          description: 'A callsign, as used in broadcasting and radio communications to identify people'
-        honorific_prefix:
-          type: string
-          description: An honorific prefix preceding a Person's name such as Dr/Mrs/Mr.
-        honorific_suffix:
-          type: string
-          description: An honorific suffix preceding a Person's name such as M.D. /PhD/MSCSW.
+      type: string
+      description: 'Describes the name of a person in format: ./{honorific prefix}/{given_name}/{family_name}/{additional_name}/{call_sign}/{honorific_suffix}'
```

Comment:
This will have to wait until the review

Reply:
Can you please change the components of name in the following order?
1. given_name (required and default)
2. honorific_prefix
3. first_name
4. middle_name
5. last_name
6. honorific_suffix

Remove additional name and call sign for now

Reply:
Done